### PR TITLE
fix(init-options): mutate forest admin client options using liana options

### PR DESCRIPTION
**Pull request by Thenkei**
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

File: src/context/build-services.js [hunk 1]
```diff
@@ -4,9 +4,9 @@ const createForestAdminClient = require('@forestadmin/forestadmin-client').defau
 module.exports = (context) =>
   context
     .addInstance('logger', () => require('../services/logger'))
-    .addUsingFunction('forestAdminClient', ({ env, logger }) => createForestAdminClient({
-      forestServerUrl: env.FOREST_URL,
+    .addUsingFunction('forestAdminClient', ({ env, forestUrl, logger }) => createForestAdminClient({
       envSecret: env.FOREST_ENV_SECRET,
+      forestServerUrl: forestUrl,
       logger: (level, ...args) => (env.DEBUG ? logger[level.toLowerCase()](...args) : {}),
     }))
```
Comment (Thenkei) [inline src/context/build-services.js:11]: I did try to make the trick using references but the `createForestAdminClient` copy the options object so we need to hard-set the `options.envSecret` directly in the `forestAdminClient` instance...

File: src/context/build-values.js [hunk 2]
```diff
@@ -9,4 +9,4 @@ module.exports = (context) =>
         : 'production',
       APPLICATION_URL: process.env.APPLICATION_URL || `http://localhost:${process.env.APPLICATION_PORT || 3310}`,
     }))
-    .addValue('forestUrl', process.env.FOREST_URL || 'https://api.forestadmin.com');
+    .addValue('forestUrl', ({ env }) => env.FOREST_URL);
```
Comment (DayTF) [inline src/context/build-values.js:12]: Why remove the default value of `https://api.forestadmin.com ` ?

**Review by DayTF — APPROVED** [review]

File: src/context/build-values.js (hunk 2 above)
Comment (DayTF) [reply src/context/build-values.js:12]: it is set just up in the 'env' object, nevermind

Comment (Thenkei) [reply src/context/build-values.js:12]: `FOREST_URL: process.env.FOREST_URL || 'https://api.forestadmin.com',`

**Pull request merged**